plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/ax_sweeper.py

```diff
@@ -0,0 +1,272 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+from dataclasses import dataclass
+from typing import Dict, List, Mapping, Optional, Sequence, Tuple, Union
+
+from ax import ParameterType  # type: ignore
+from ax.core import types as ax_types  # type: ignore
+from ax.service.ax_client import AxClient  # type: ignore
+from omegaconf import DictConfig, OmegaConf
+
+from hydra.core.config_loader import ConfigLoader
+from hydra.core.config_search_path import ConfigSearchPath
+from hydra.core.plugins import Plugins
+from hydra.plugins.launcher import Launcher
+from hydra.plugins.search_path_plugin import SearchPathPlugin
+from hydra.plugins.sweeper import Sweeper
+from hydra.types import TaskFunction
+
+from .earlystopper import EarlyStopper
+
+log = logging.getLogger(__name__)
+
+
+@dataclass
+class Trial:
+    overrides: List[str]
+    trial_index: int
+
+
+BatchOfTrialType = List[Trial]
+# ParameterType = Union[ax_types.TParameterization]
+
+# TODO: output directory is overwriting, job.num should be adjusted (depends on issue #284)
+# TODO: Support running multiple random seeds, aggregate mean and SEM
```

Comment:
I think we can remove this  TODO unless we actually want to do it. in that case there should be a task.

Reply:
Removing for now.